src/plugins/intel_cpu/src/nodes/reduce.cpp

```diff
@@ -2274,6 +2274,16 @@ void Reduce::execute(dnnl::stream strm) {
     const uint8_t *src_data = srcMemPtr->getDataAs<const uint8_t>();
     uint8_t *dst_data = dstMemPtr->getDataAs<uint8_t>();
 
+    const auto& src_shape = srcMemPtr->getStaticDims();
+    empty_input = shape_size(src_shape) == 0 || srcMemPtr->getSize() == 0;
+    if (empty_input) {
+        if (dstMemPtr->getSize() > 0) {
+            init_dst_data(dst_data, dstMemPtr->getSize());
+            reduce_kernel_post_process(dst_data);
+        }
```

Comment:
Is the previously added check to skip post process if `attr.get()->post_ops_.len() == 0` redundant?

Reply:
JIT kernel will check post ops len and skip anyway if it is 0. But you are right, skip earlier is better. Applied. Thanks Katarzyna!